Sources/EduIDExpansion/Flows/Activity/ActivityCoordinator.swift

```diff
@@ -0,0 +1,27 @@
+import UIKit
+
+class ActivityCoordinator: CoordinatorType {
+    
+    weak var navigationController: UINavigationController?
+    weak var delegate: ActivitMainNavigationDelegate?
+    
+    func start(presentOn viewController: UIViewController) {
```

Comment:
Normally we pass in the viewController during coordinator init.

Reply:
the viewController to present on?

Reply:
Yes.

Reply:
I did this